### PR TITLE
libverto: add libverto/0.3.2 recipe

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **libverto/0.3.2**

Requirement of krb5

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [X] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by SSE4 — DISMISSED** [review]

File: recipes/libverto/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+            raise ConanInvalidConfiguration("Default backend({}) must be available".format(self.options.default))
+
+        count = lambda iterable: sum(1 if it else 0 for it in iterable)
+        count_builtins = count(str(opt) == "builtin" for opt in self._backend_dict.values())
+        count_externals = count(str(opt) == "external" for opt in self._backend_dict.values())
+        if count_builtins > 1:
+            raise ConanInvalidConfiguration("Cannot have more then one builtin backend")
+        if not self.options.shared:
+            if count_externals > 0:
+                raise ConanInvalidConfiguration("Cannot have a non-builtin backend when building a static libverto")
```
Comment (prince-chrismc) [inline recipes/libverto/all/conanfile.py:89]:
```suggestion
                raise ConanInvalidConfiguration("Cannot have an external backend when building a static libverto")
```

File: recipes/libverto/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+                    yield
+        else:
+            yield
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
+        self._autotools.libs = []
+        if self.settings.compiler == "Visual Studio":
```
Comment (prince-chrismc) [inline recipes/libverto/all/conanfile.py:134]: This is blocked 🤔

File: recipes/libverto/all/test_package/test_package.c [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    }
+}
+
+int main(int argc, char *argv[])
+{
+    verto_ctx *ctx;
+    verto_ev *ev;
+
+    if (argc < 2) {
+        fprintf(stderr, "Need an argument\n");
```
Comment (prince-chrismc) [inline recipes/libverto/all/test_package/test_package.c:30]:
```suggestion
        fprintf(stderr, "Need to pass the backend as an argument\n");
```

**Review by prince-chrismc — COMMENTED** [review]
two very minor suggestions

File: recipes/libverto/all/conanfile.py [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+            self._autotools.flags.append("-FS")
+            self._autotools.link_flags.extend("-L{}".format(p.replace("\\", "/")) for p in self.deps_cpp_info.lib_paths)
```
Comment (madebr) [inline recipes/libverto/all/conanfile.py:136]:
```suggestion
```

**Review by prince-chrismc — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**